Fastlane/deployment_lanes.rb

```diff
@@ -292,8 +292,10 @@
     success_message = "A new #{release_type} has been submitted to the App Store."
     UI.success "#{success_message} (#{tag_name})"
     slack_message(success_message, type: :submitted, tag_name: tag_name, release_url: release_url)
-  rescue StandardError => e
-    UI.error e
+
+  rescue StandardError => exception
+    UI.error exception
+    handle_error(:release, exception)
```

Comment:
Does this method handle removing tags and ...?

Reply:
Not yet? Should it? I guess that would be nice.. we shall create a ticket for that :)

Reply:
Yeah, I think it should, because if we run the workflow again we may have conflicts

Reply:
@peagasilva
Could we use `clean_up_release_from_tag` here?
At the moment `handle_error` only calls it, if the lane is `release_from_tag`.

Reply:
@neodym I would say so - we need to only update the filter. Sorry for the delay, I totally missed this message.